index.html

```diff
@@ -1,67 +1,345 @@
-          </div>
-        </a>
-        <div class="toolbar-container">
-          
         </div>
-      </div>
-    </nav>
-
-    </div>
-  </header>
-  <main>
-    <section class="headline">
-      <div class="primary-text-2">
-        <h1>I'm Zermarden <br>
-        Glad to see <br>
-        you!</h1>
-      </div>      
-      <div class="social-media-icons">
-        <ul class="social-media-icons-list">
-          <li><a href="https://twitter.com/Zermarden"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
-          <li><a href="https://www.linkedin.com/in/zermarden/"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
-          <li><a href="https://medium.com/@zermarden"><img src="assets/m.svg" alt="Medium" width="20" height="20"></a></li>
-          <li><a href="https://github.com/Nedramrez"><img src="assets/git.svg" alt="GitHub" width="20" height="20"></a></li>
-          <li><a href="https://angel.co/u/zermarden-orazgaliyev"><img src="assets/finger.svg" alt="Peace" width="20" height="20"></a></li>
-      </ul>
-      </div>
-      <div class="social-media">
-        <div class="supporting-text">
-          <span>LET'S CONNECT!</span>
+        <section class="headline">
+          <h1 class="primary-text-2">I'm Zermarden <br>
+          Glad to see <br>
+          you!</h1>
+          <p class="primary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <div class="social-media">
+            <div class="supporting-text">
+              <span>LET'S CONNECT!</span>
+            </div>
+          </div>
+          <div class="social-media-icons">
+            <ul class="social-media-icons-list">
+              <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+              <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+              <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
+              <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
+              <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
+          </ul>
+          </div>
+        </section>
+        <img class="header-shapes" src="assets/bg.png" alt="header-shapes">
+        <img class="header-background" src="assets/header-background-mobile.svg" alt="background-image">
+    </header>
+    <main>
+        <div class="works">
+          <div class="card-works-1">
+            <div class="snapshot-portfolio">
+              <img class="snp-p-img" src="assets/tonic.svg" width="295" height="220" alt="shapshot-portfolio">
+            </div>
+            <div class="left-block">
+              <div class="primary-text-3">
```

Comment:
- [ ] Please kindly make the container of the `works` section a `<section>` tag instead of a `<div>` in order to have semantically correct HTML 👍